libs/api/domains/education/src/lib/education.service.ts

```diff
@@ -66,13 +66,25 @@ export class EducationService {
     })
   }
 
-  async getFamily(nationalId: string) {
+  private isChild(familyMember: ISLFjolskyldan): boolean {
+    return (
+      !['1', '2', '7'].includes(familyMember.Kyn) &&
```

Comment:
Would be nice to use enum values here for readability.

Reply:
agree.. or honestly, just using the new national registry service instead.